src/rdf-repository/grafter/rdf/sparql.clj

```diff
@@ -3,27 +3,45 @@
             [grafter.rdf.repository :refer [repo sparql-repo ->connection]]
             [grafter.rdf.repository :as repo]
             [grafter.rdf.io :refer [->sesame-rdf-type]]
-            [clojure.java.io :refer [resource]]))
+            [clojure.java.io :refer [resource]]
+            [clojure.string :as str]))
+
+(defn pre-process-limit-clauses
```

Comment:
Only just spotted this one, and I'm happy to clean it up when I merge; but I think we should make this private with a `defn-`.  

If you still want to test it you can access "private" vars by looking them up via the var instead of the symbol e.g. something like this:

`(#'my.private/function 1 2 3)`